src/vochain/vochain.proto

```diff
@@ -39,6 +54,20 @@ message ProofIden3 {
         bytes siblings = 1;
 }
 
+message ProofEthereumStorage {
+        bytes key = 1;
+        bytes value = 2;
+        repeated bytes siblings = 3;
+}
+
+message ProofEthereumAccount {
+        bytes nonce = 1;
+        bytes balance = 2;  // Big Int encoded as bytes
+        bytes storageHash = 3;
+        bytes codeHash = 4;
+        repeated bytes siblings = 5;
+}
+
```

Comment:
This 'change' is just putting the Proofs in the same order that appear at the 'message Proof' payload declaration at line 32-45.